.ci/coverage.py

```diff
@@ -0,0 +1,13 @@
+from lcov_cobertura import LcovCobertura
```

Comment:
Add shebang: 
#!/usr/bin/env python3

Reply:
Added, is this a subtle way of suggesting we use all python3? :-D

Reply:
Python2 is retiring in 14 months, https://pythonclock.org/